pkg/logs/input/docker/scanner.go

```diff
@@ -144,21 +144,25 @@ func (s *Scanner) setup() error {
 	}
 
 	// Start tailing monitored containers
-	s.scan(false)
+	s.scan(true)
 	return nil
 }
 
 // setupTailer sets one tailer, making it tail from the beginning or the end,
 // returns true if the setup succeeded, false otherwise
-func (s *Scanner) setupTailer(cli *client.Client, container types.Container, source *config.LogSource, tailFromBeginning bool, outputChan chan message.Message) bool {
-	log.Info("Detected container ", container.Image, " - ", s.humanReadableContainerID(container.ID))
-	t := NewTailer(cli, container.ID, source, outputChan)
-	err := t.recoverTailing(s.auditor, tailFromBeginning)
+func (s *Scanner) setupTailer(cli *client.Client, container types.Container, source *config.LogSource, isSetup bool, outputChan chan message.Message) bool {
+	log.Info("Detected container ", container.Image, " - ", ShortContainerID(container.ID))
+	tailer := NewTailer(cli, container.ID, source, outputChan)
+	since, err := Since(s.registry, tailer.Identifier(), isSetup)
```

Comment:
`isSetup` is confusing, we don't know if it is about the tailer or the agent, or the scanner